ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -482,9 +491,11 @@ export default defineComponent({
 
       // If transfer is valid, enter review mode
       if (localState.isValidTransfer) {
+        setValidation(MhrSectVal.ADD_EDIT_OWNERS_VALID, MhrCompVal.OWNERS_VALID, true)
         localState.isReviewMode = true
         localState.validate = false
       } else {
+        setValidation(MhrSectVal.ADD_EDIT_OWNERS_VALID, MhrCompVal.OWNERS_VALID, isGlobalEditingMode.value)
```

Comment:
MHR Information flow and MH Registrations don't share a validation state, so i'm a little curious about this implementation. 
I don't believe we should need to change the MHR Validation state in our MHR Information flow. 

If you don't mind, perhaps look a bit deeper into WHY you needed to do this. I feel like the `isGlobalEditingMode` flag should* be enough to represent the state of the add edit component. Saving this state back to another validation state doesn't seem quite right 🤔

Reply:
Way cleaner now, thanks for the chat:) @cameron-eyds

Reply:
Awesome! No prob at all sir 👍